pkg/fixtures/writer_plain_mysql.go

```diff
@@ -0,0 +1,78 @@
+package fixtures
+
+import (
+	"fmt"
+	"github.com/Masterminds/squirrel"
+	"github.com/applike/gosoline/pkg/cfg"
+	"github.com/applike/gosoline/pkg/db"
+	"github.com/applike/gosoline/pkg/mon"
+	"reflect"
+)
+
+type MySqlPlainFixtureValues []interface{}
+
+type MySqlPlainMetaData struct {
+	TableName string
+	Columns   []string
+}
+
+type mySqlPlainFixtureWriter struct {
+	logger   mon.Logger
+	dbClient db.Client
+	metaData MySqlPlainMetaData
+}
+
+func MySqlPlainFixtureWriterFactory(metaData MySqlPlainMetaData) FixtureWriterFactory {
+	return func(config cfg.Config, logger mon.Logger) FixtureWriter {
+		dbClient := db.NewClient(config, logger)
+		return NewMysqlPlainFixtureWriterWithInterfaces(logger, dbClient, metaData)
+	}
+}
+
+func NewMysqlPlainFixtureWriterWithInterfaces(logger mon.Logger, dbClient db.Client, metaData MySqlPlainMetaData) FixtureWriter {
+	return &mySqlPlainFixtureWriter{
+		logger:   logger,
+		dbClient: dbClient,
+		metaData: metaData,
+	}
+}
+
+func (m *mySqlPlainFixtureWriter) Write(fs *FixtureSet) error {
+	for _, item := range fs.Fixtures {
+		fixture, ok := item.(*MySqlPlainFixtureValues)
+
+		if !ok {
+			return fmt.Errorf("invalid fixture type: %s", reflect.TypeOf(item))
+		}
+
+		sql, args, err := m.buildSql(fixture)
+
+		if err != nil {
+			return err
+		}
+
+		res, err := m.dbClient.Exec(sql, args...)
+
+		if err != nil {
+			return err
+		}
+
+		ar, err := res.RowsAffected()
+
+		if err != nil {
+			return err
+		}
+
+		m.logger.Info(fmt.Sprintf("affected rows while fixture loading: %d", ar))
+	}
+
+	m.logger.Infof("loaded %d plain mysql fixtures", len(fs.Fixtures))
+
+	return nil
+}
+
+func (m *mySqlPlainFixtureWriter) buildSql(values *MySqlPlainFixtureValues) (string, []interface{}, error) {
+	insertBuilder := squirrel.Replace(m.metaData.TableName).PlaceholderFormat(squirrel.Question).Columns(m.metaData.Columns...).Values(*values...)
```

Comment:
from the last commits it looks like logic changed a bit, but i guess you checked it still works properly?
I would like to see this line to become a bit shorter, how about this:
```suggestion
	insertBuilder := squirrel.Replace(m.metaData.TableName).
	                          PlaceholderFormat(squirrel.Question).
	                          Columns(m.metaData.Columns...).
	                          Values(*values...)
```